content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -98,15 +98,17 @@ This feature is useful for day-to-day operational tasks, such as debugging a stu
 
 ![Image of the Pulumi cloud console actions](./pulumi-cloud-actions.png)
 
-<!-- TODO: show actions > destroy UI -->
+### Destroy all stack resources
+
+After you are finished exploring your new stack, you can begin to clean up the resources created. In the **Actions** tab, select the **Destroy** option to remove all resources associated with the stack. Confirm by clicking the **Delete resources?** confirmation box, then click **Deploy**. Pulumi will begin tearing down the resources, and you can track the progress directly in the console.
+
+If you no longer need the stack itself, you can also delete it. In the same **Settings** tab, choose **Delete stack** to remove the stack, including all its deployment history and state files from Pulumi Cloud.
 
 ## Cleanup
```

Comment:
There's no cost for having public repos on GitHub, so I'd probably just delete this section or pull the bit about deleting the repo into the section above this one, and maybe call that "Clean Up" instead (which is where we usually put destroy steps like these). (Which reminds me -- is it possible to create a new project with a _private_ repo? 🤔 We should probably call out that repos are public by default, or perhaps they're in line with your organization's default repo-creation settings?)

Reply:
In my testing its not something you can set in the pulumi UI, it defaults to my settings, in this case I happened to create a private org, so they've all been private.